### DISCUSSION TITLE
allow Airdrop to transfer files

### DISCUSSION THREAD

**Discussion by mfarcas-it** [usnistgov/macos_security#127] [Q&A]
Hola :) 
After pushing a configuration profile based on os_airdrop_disable.yaml, someone decided to exclude a few users from the scope.
Upon removing the configuration profile on my test devices, I could start Airdrop again, but I cannot find my device to send a file too. I can see my phone but not my laptop. Any ideas on what needs to be changed in order for the laptop to be in discovery mode? Note: that my airdrop is set to be discoverable by everyone.
Many thanks!

**Comment by erefneb** [comment]
Make sure both wifi and Bluetooth are both active on all devices involved.

Also… https://support.apple.com/en-us/HT203106

--
Eric Benfer
***@***.***

**Reply by mfarcas-it** [reply]
Both of them are active :)
